var/spack/repos/builtin/packages/libffs/package.py

```diff
@@ -38,16 +38,21 @@ class Libffs(CMakePackage):
 
     version('develop', git='https://github.com/GTkorvo/ffs.git',
             branch='master')
+    version('1.5', 'c41c5f5f448b627740deecd695b7bbf8')
     version('1.1.1', 'aa1c8ad5cf35e8cf76735e3a60891509')
     version('1.1', '561c6b3abc53e12b3c01192e8ef2ffbc')
 
-    depends_on('flex')
-    depends_on('bison')
+    depends_on('flex', type='build', when='@:1.1.1')
+    depends_on('bison', type='build', when='@:1.1.1')
+    depends_on('gtkorvo-cercs-env', type='build', when='@:1.1.1')
```

Comment:
Do you know if these are the exact versions where these dependencies were dropped? It would be nice to be specific in case someone adds another versions between 1.1.1 and 1.5.

Reply:
Everything prior to 1.5 requires flex, bison and cercs-env.

Reply:
In that case let's do `when='@:1.4'`